### ISSUE TITLE
About this dataset

### ISSUE THREAD

**Issue by shahzad-ali** [uwm-bigdata/wound-segmentation#4]
Hi @uwm-bigdata, @Pele324 

Thanks for shairng this dataset with the whole research community 👏🏼

1. What's the proper way to cite [Foot Ulcer Segmentation Challenge](https://github.com/uwm-bigdata/wound-segmentation/tree/master/data/Foot%20Ulcer%20Segmentation%20Challenge) dataset?
2. Can you list some recently published papers that made use of it?

Looking forward to hearing from you.

**Comment by Pele324** [comment]
> Hi @uwm-bigdata, @Pele324
> 
> Thanks for shairng this dataset with the whole research community 👏🏼
> 
> 1. What's the proper way to cite [Foot Ulcer Segmentation Challenge](https://github.com/uwm-bigdata/wound-segmentation/tree/master/data/Foot%20Ulcer%20Segmentation%20Challenge) dataset?
> 2. Can you list some recently published papers that made use of it?
> 
> Looking forward to hearing from you.

Hi @shahzad-ali   Thanks for your interest in the dataset! The challenge dataset "evolves" from our previous work(https://doi.org/10.1038/s41598-020-78799-w). Please cite this paper. Other than that we are not aware of publications making use of the data.

**Comment by shahzad-ali** [comment]
Thanks for your reply!

According to your paper "[_Fully automatic wound segmentation with deep convolutional neural networks_](https://doi.org/10.1038/s41598-020-78799-w)", the dataset has total of 1,109 foot ulcer images but the released data has 610 training + 400 validation = 1,010 images. It seems likely that the current dataset will undergo many changes so using a labeling system like `release version` to distinguish different releases will make sense.

There are two challenges `FUS (Foot ulcer segmentation)` and `DFU (Diabetic Foot Ulcer)`. Earlier one is not the part of MICCAI 2021 and later one is missing the segmentation data. Is this correct? 

Should we expect publicly released segmentation data under FUS 2021 Challenge?

**Comment by Pele324** [comment]
> Thanks for your reply!
> 
> According to your paper "[_Fully automatic wound segmentation with deep convolutional neural networks_](https://doi.org/10.1038/s41598-020-78799-w)", the dataset has total of 1,109 foot ulcer images but the released data has 610 training + 400 validation = 1,010 images. It seems likely that the current dataset will undergo many changes so using a labeling system like `release version` to distinguish different releases will make sense.

Thanks for the suggestion! You are right we are working with our cooperating clinic and continuously updating the dataset by refining current data/adding new data. Consider the challenge dataset as ver1.0 :smile:

> There are two challenges `FUS (Foot ulcer segmentation)` and `DFU (Diabetic Foot Ulcer)`. Earlier one is not the part of MICCAI 2021 and later one is missing the segmentation data. Is this correct?

Yes, our FUSeg is a newly published open challenge that is not tied to MICCAI 2021 like DFU. FUSeg focuses on segmentation while DFU focuses on detection and localization. (I think DFU is planning to include segmentation in the future)

> Should we expect publicly released segmentation data under FUS 2021 Challenge?

The ground truth masks are already published in this repo if this is what you mean. For example, in our [train folder](https://github.com/uwm-bigdata/wound-segmentation/tree/master/data/Foot%20Ulcer%20Segmentation%20Challenge/train) you can find both the images and labels(ground truth masks).
